core/templates/domain/story/StoryObjectFactorySpec.ts

```diff
@@ -43,18 +43,24 @@ describe('Story object factory', () => {
         nodes: [{
           id: 'node_1',
           title: 'Title 1',
+          description: 'Description',
           prerequisite_skill_ids: [],
           acquired_skill_ids: [],
           destination_node_ids: [],
           outline: 'Outline',
           exploration_id: null,
-          outline_is_finalized: false
+          outline_is_finalized: false,
+          thumbnail_filename: 'img.png',
+          thumbnail_bg_color: '#a33f40'
         }],
         next_node_id: 'node_3'
       },
       language_code: 'en'
     };
     _sampleStory = storyObjectFactory.createFromBackendDict(
+      // TS ignore is used because sample story doesn't have thumbail to test
```

Comment:
@aks681 @kevintab95 is this actually an issue that needs to be fixed, and not ignored?

Reply:
Also, @nishantwrp is this because the filename you have given doesn't exist? I think you could add an image and use that file name....

Reply:
No actually, there should be a thumbnail in the `sampleStoryBackendDict` but I didn't add that because this test tests those validations.